app/redux/review.js

```diff
@@ -1,28 +1,10 @@
 import axios from 'axios'
 
-const SET_REVIEW = 'SET_REVIEW'
-
-const setReview = (review) => {
-    return {
-        type: SET_REVIEW,
-        review
-    }
-}
-
-export const reviewReducer = (state = [], action) => {
-    switch (action.type) {
-        case SET_REVIEW:
-            return action.review;
-        default:
-            return state;
-    }
-}
-//thunk to fetch reviews
-
-export const fetchReviews = () => {
+//thunk to add a review
+export const addReview = (review, productId) => {
     return (dispatch) => {
-        axios.get('/api/review')
-            .then(reviews => dispatch(setReview(reviews.data)))
+        axios.post('/api/review', (review))
+            .then(review => dispatch(addReview(review, productId)))
```

Comment:
pretty close - we're not dispatching the review, though, we're just posting it. so the post request should look like:

`axios.post('/api/review/:productId', review)`

the `.then` statement can just be a dispatch of `fetchProducts`, with no arguments passed in (it will have to be imported from `/redux/products`)